### ISSUE TITLE
Does it support matomo?

### ISSUE THREAD

**Issue by kristian2x** [joernroeder/piwik-react-router#60]
I am confused by piwiks name change. Does this support matomo? If yes is it only supported for react-router v5? Related comment/issue: https://github.com/joernroeder/piwik-react-router/issues/58#issuecomment-520240136

**Comment by mattab** [comment]
Yes it should support Matomo. It would be awesome to rename this project, maybe someone can open a PR or fork the repo with the new name? I think maybe this project is not maintained anymore. We'd be happy to link to the new repo and project from https://matomo.org/integrate/

**Comment by idodav** [comment]
I've found a newer project. It is currently in maintanace. It was last updated at the end of 2019.
[guillaumeparis2000/React-Piwik](https://github.com/guillaumeparis2000/react-piwik)
 Please link the newer project in your website.
